edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -136,7 +136,7 @@ var edx = edx || {};
         updateRemainingTime: function (self) {
             self.timerTick ++;
             self.secondsLeft --;
-            if (self.timerTick % 30 === 0){
+            if (self.timerTick % 60 === 0){
```

Comment:
Nit: I think it would be clearer if this was a constant that was named as to its purpose.